resume-input/src/Components/Basics/Basics.jsx

```diff
@@ -8,36 +8,56 @@ export default function Basics(props) {
 
   return (
     <div className="basic">
-      <div>
-        <label className="" htmlFor="name">

```

Comment:
I see that there are a few empty classNames is there a reason for this
I am still new to programming, so I am actually asking for insight not as a critique :)
Thanks